mlrun/api/api/endpoints/functions.py

```diff
@@ -615,6 +615,12 @@ def _build_function(
                         # initialize model monitoring stream
                         _create_model_monitoring_stream(project=fn.metadata.project)
 
+                        if not fn.spec.tracking_policy:
+                            # initialize tracking policy with default values
+                            fn.spec.tracking_policy = (
+                                mlrun.utils.model_monitoring.TrackingPolicy().to_dict()
```

Comment:
```suggestion
                                mlrun.utils.model_monitoring.TrackingPolicy()
```
Here you're putting a dictionary in the `spec.tracking_policy` field. On the client-side code you're using the `TrackingPolicy` class. This needs consistency - I believe the correct thing would be to use the class, meaning that you should not use `to_dict()` here.